src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,21 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+
+# Configure chocolatey to use FireEye's package repository
+iex "choco sources add -n=fireeye -s https://www.myget.org/F/fireeye/api/v2 --priority 1"
+iex "choco feature enable -n allowGlobalConfirmation"
+iex "choco feature enable -n allowEmptyChecksums"
+
+& powercfg -change -monitor-timeout-ac 0 | Out-Null
```

Comment:
Some explanatory commenting would be good to document the script's functionality.

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/2a210e164b2509f876d3afb61de66747e5669e27